src/Control/Distributed/Process/Internal/Types.hs

```diff
@@ -345,7 +350,37 @@ data LocalProcessState = LocalProcessState
 newtype Process a = Process {
     unProcess :: ReaderT LocalProcess IO a
   }
-  deriving (Functor, Monad, MonadIO, MonadReader LocalProcess, Typeable, Applicative, MonadFix)
+  deriving ( Applicative, Functor, Monad, MonadFix, MonadIO
```

Comment:
minor style point: if you're going to introduce a line break here, one class name per line will make it more manageable.